week2/02_mlflow_experiment.py

```
@@ -0,0 +1,54 @@
+import subprocess
+import mlflow
+
+mlflow.set_tracking_uri("databricks")
+
+mlflow.set_experiment(experiment_name="/Shared/life-expectancy-basic")
+mlflow.set_experiment_tags({"repository_name": "life-expectancy"})
+
+def get_git_sha():
+    try:
+        return subprocess.check_output(['git', 'rev-parse', 'HEAD']).decode('ascii').strip()[:7]
+    except:
```

Ruff (0.7.0)

Do not use bare `except`

(E722)